tracker.json

```diff
@@ -1,8 +1,8 @@
 {
   "fire_protetion_hu": {
-    "version": "0.0.2",
-    "updated_at": "2022-09-29",
-    "visit_repo": "https://github.com/amaximus/fire_protection_hu",
-    "changelog": "https://github.com/amaximus/fire_protection_hu/releases/0.0.2"
+    "version": "0.0.2a",
+    "updated_at": "2022-10-21",
+    "visit_repo": "https://github.com/gtgt/ha-fire_protection_hu",
```

Comment:
Since it's a pull request you should leave the visit_repo and change log to point to the original repo.
If you want your own version with your enhancement then it should not be a pull request, but a simple fork and then you can use references to your repo, but you'll have to add yourself your repo to HACS